cmake/toolchain/zephyr/host-tools.cmake

```diff
@@ -15,7 +15,7 @@ if(NOT ZEPHYR_SDK_INSTALL_DIR)
   return()
 endif()
 
-set(REQUIRED_SDK_VER 0.9.5)
+set(REQUIRED_SDK_VER 0.10.0)
```

Comment:
actually, lets give people more time with this, otherwise it will break downstream CI